src/Resources/laravel-debugbar.css

```diff
@@ -579,6 +597,10 @@ ul.phpdebugbar-widgets-list li.phpdebugbar-widgets-list-item .phpdebugbar-widget
     max-width: 100%;
 }
 
+ul.phpdebugbar-widgets-list li.phpdebugbar-widgets-list-item {
+    line-height: 20px;
```

Comment:
After this change seems like messages list grows too much on gate tab, and the icons are misaligned
![image](https://github.com/barryvdh/laravel-debugbar/assets/4933954/9311b75a-7498-4dd4-a680-24bbd739073b)
Same on messages tab
![image](https://github.com/barryvdh/laravel-debugbar/assets/4933954/566af5e7-b827-49a0-aadc-eee76220d0c1)

Why don't just normal line height??

Reply:
Try #1604